src/main/java/frc/robot/constants/VisionConstants.java

```diff
@@ -105,11 +105,30 @@ public final class DoryCameras {
                 Units.degreesToRadians(-147)));
   }
 
+  /** Standard Deviations for when we see a single target with a high resolution camera. */
   public static final Matrix<N3, N1> highResSingleTagStdDev =
       VecBuilder.fill(0.4, 0.4, Double.MAX_VALUE);
+
+  /** Maximum distance to trust data when we see a single target with a high resolution camera. */
+  public static final double highResSingleTagMaxDist = 5; // TODO get new value
+
+  /** Standard Deviations for when we see a single target with a normal resolution camera. */
   public static final Matrix<N3, N1> normalSingleTagStdDev =
       VecBuilder.fill(0.8, 0.8, Double.MAX_VALUE);
+
+  /** Maximum distance to trust data when we see a single target with a normal resolution camera. */
+  public static final double normalSingleTagMaxDist = 5; // TODO get new value
+
+  /** Standard Deviations for when we see multiple targets with a high resolution camera. */
```

Comment:
Clarify these comments to indicate it's a *reference* standard deviation at one meter.